core/lib/libtomcrypt/sm2-dsa.c

```diff
@@ -18,10 +18,9 @@
 /*
  * GM/T 0003.1‒2012 Part1 2 Section 6.1
  */
-TEE_Result crypto_acipher_sm2_dsa_sign(uint32_t algo,
-				       struct ecc_keypair *key,
-				       const uint8_t *msg, size_t msg_len,
-				       uint8_t *sig, size_t *sig_len)
+TEE_Result ltc_sm2_dsa_sign(uint32_t algo, struct ecc_keypair *key,
```

Comment:
Same here, with the `ltc_` prefix this sounds too much like an "official" LTC function.

Reply:
ok, I'll change to `sm2_ltc_...`